cranelift/codegen/src/value_label.rs

```diff
@@ -23,7 +19,6 @@ pub struct ValueLocRange {
 
 /// The particular location for a value.
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
-#[cfg_attr(feature = "enable-serde", derive(Serialize, Deserialize))]
```

Comment:
Why did you remove those?

Reply:
Ah, regalloc2 currently doesn't have serde support so I removed this to get things building, and then forgot to come back to it later... I'll add this.

Reply:
Fixed!